lessons/flow.md

```diff
@@ -0,0 +1,132 @@
+
+
+# Modern JS workshop - Flow
+
+## Table of contents
+1. [What is Flow?](#1-what-is-flow)
+2. [Getting started](#2-getting-started)
+   * [Setup the compiler](#setup-the-compiler)
+   * [Setup Flow client](#setup-flow-client)
+   * [Prepare your files for Flow](#prepare-your-files-for-flow) 
+3. [Writing Flow code](#3-writing-flow-code)
+4. [Where do we use it at The Guardian?](#4-where-do-we-use-it-at-the-guardian)
+5. [Resources](#5-resources)
+
+
+## 1. What is Flow?
+
+According to Flow's site:
+
+> Flow is a static type checker for javascript
+
+
```

Comment:
I wonder whether here you could expand on "static" saying that flow gives errors *while* developing, but in order for it to run in a browser you need to remove the types from the javascript code. Then run into removing the types? Maybe that's too simplistic?? I dunno :man_shrugging:

Reply:
Totally agree, this first PR was more focused on the second point of this document (Getting started). I will do another PR to expand the other sections.

Reply:
I thought it might be, sounds good!